### PR TITLE
Fix Bug 1136660 - Make product wordmarks high resolution

### PR DESCRIPTION
**Pull request by kyoshino**
Images are optimized.

### CODE REVIEW

**Comment by kyoshino** [comment]
Some images might not be necessary since the Firefox family page redesign will replace those anyway. I could also find the retina Tabzilla with the new Mozilla wordmark in a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1135092).

File: media/css/mozorg/home/home.less [hunk 1]
```diff
@@ -1296,3 +1292,15 @@ h1, h2, h3, h4, h5, h6 {
         }
     }
 }
+
+/****************************************************************************/
+// @High resolution images for @retina-type displays
+// Desktop
+@media (-webkit-min-device-pixel-ratio: 1.5),
+       (                min-resolution: 1.5dppx),
+       (                min-resolution: 144dpi) {
```
Comment (sgarrity) [inline media/css/mozorg/home/home.less]: Could we use a mixin to setup these media queries?

Comment (kyoshino) [inline media/css/mozorg/home/home.less]: Thanks, nice mixin!

**Comment by kyoshino** [comment]
I guess no one is working on Firefox family pages yet, then it might be okay to get this merged.

**Comment by cargoworker** [comment]
Can one of the admins verify this patch?

File: media/css/tabzilla/tabzilla.less [hunk 2]
```diff
@@ -621,6 +621,12 @@ html[dir="rtl"] {
        (                min-resolution: 1.5dppx),
        (                min-resolution: 144dpi) {
 
+    // high res tab
+    #tabzilla {
+        background-image: url('/media/img/tabzilla/tab-high-res.png');
+        background-size: 150px 90px;
```
Comment (alexgibson) [inline media/css/tabzilla/tabzilla.less]: Nit: we should add a `-webkit` vendor prefix here to `background-size`

**Comment by alexgibson** [comment]
Firefox wordmark on the home page is egged at smaller viewport sizes

egged-logo

**Comment by alexgibson** [comment]
I think some of the images here could be optimized a bit more perhaps? 

For example, I was still able to run `title-beta-high-res.png` through ImageOptim which saved another 14%. I then tried some more lossy compression with TinyPNG, which then saved a further 33%.

Lossy compression might not work as well for some of the more detailed images, but I think for simple word-marks it should be fine. Still worth trying for most images I think, as the file size savings can be considerable :smile:

File: media/css/tabzilla/tabzilla.less (hunk 2 above)
Comment (craigcook) [inline media/css/tabzilla/tabzilla.less]: Use the <code>.background-size()</code> mixin as a shortcut.

Comment (alexgibson) [inline media/css/tabzilla/tabzilla.less]: Good call @craigcook - I forgot `lib.less` was referenced in Tabzilla's CSS :+1:

Comment (kyoshino) [inline media/css/tabzilla/tabzilla.less]: Somehow I thought I couldn't use the mixin in `tabzilla.less `. Will use it :+1:

**Comment by kyoshino** [comment]
Thank you @alexgibson! I'll fix the issues later.

**Comment by kyoshino** [comment]
Fixed the Alien's egg issue and further optimized wordmarks using TinyPNG :smiley_cat:

**Comment by alexgibson** [comment]
This looks good to me, r+ thanks @kyoshino (sorry this one got put on the back burner longer than it should!)

**Comment by kyoshino** [comment]
Thanks!

**Pull request merged**